core/new-gui/src/app/workspace/service/execute-workflow/execute-workflow.service.ts

```diff
@@ -173,9 +179,11 @@ export class ExecuteWorkflowService {
     // get the current workflow graph
     const logicalPlan = ExecuteWorkflowService.getLogicalPlanRequest(this.workflowActionService.getTexeraGraph());
     console.log(logicalPlan);
+    var engineVersion = this.version.hash;
```

Comment:
no need for these two lines, you can just pass them below {logicalplan, version.hash}

Reply:
removed